support-frontend/assets/pages/supporter-plus-thank-you/components/thankYouHeader/heading.tsx

```diff
@@ -217,6 +220,16 @@ function Heading({
 		);
 	}
 
+	if (isDigitalEdition) {
+		return (
+			<h1 css={headerTitleText}>
+				Thank you
+				<span data-qm-masking="blocklist">{maybeNameAndCommaSpace}</span>
```

Comment:
Does this need to be different for digital edition? The only difference seems to be the commas?

Reply:
Yes, presumed needed to match as close as possible. This could not go in the block below as was highlighted yellow too. Just seemed easier for removal in future to seperate. Hope ok